lib/mumuki/laboratory/mumukit/randomizations/with_randomizations.rb

```diff
@@ -0,0 +1,32 @@
+module WithRandomizations
+  def self.included(base)
```

Comment:
Use `ActiveSupport::Concern`  and `included do ... end` instead

Reply:
I need to use the base parameter for this, and I'm not sure how to do that with ActiveSupport's included :thinking:

Reply:
nvm this. I just realized how to do it :stuck_out_tongue_closed_eyes: